src/main/java/uk/gov/companieshouse/accountsDates/impl/AccountsDatesImpl.java

```diff
@@ -0,0 +1,255 @@
+package uk.gov.companieshouse.accountsDates.impl;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.Period;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+import uk.gov.companieshouse.accountsDates.AccountsDates;
+
+public class AccountsDatesImpl implements AccountsDates {
+
+    private static final String PERIOD_START = "periodStart";
+    private static final String PERIOD_END = "periodEnd";
+    private static final String DATE_FORMAT_YYYYMMDD = "yyyy-MM-dd";
+    private static final String DATE_FORMAT_D_MMMM_YYYY = "d MMMM yyyy";
+
+    /**
+     * Takes a {@link String} date in format yyyy-MM-dd and converts it to a Java 8 {@link LocalDate}
+     * 
+     * @param stringDate
+     * @return
+     */
+    @Override
+    public LocalDate convertStringToDate(String stringDate) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMAT_YYYYMMDD);
+        
+        return LocalDate.parse(stringDate, formatter);
+    }
+    
+    /**
+     * Takes a {@link String} in "yyyy-MM-dd'T'HH:mm:ss.SSSZ" format and converts it to a Java 8 {@link LocalDate}
+     * 
+     * @param dateTimeString
+     * @return 
+     */
+    @Override
+    public LocalDate getLocalDatefromDateTimeString(String dateTimeString) {
+        Instant instant = Instant.parse(dateTimeString);
+        LocalDateTime localDateTime = LocalDateTime.ofInstant(instant, ZoneId.of(ZoneOffset.UTC.getId()));
```

Comment:
As this is a library, can you make the time zone a parameter please? This is assuming all times passed into this helper are in UTC timezone. A library shouldn't make assumptions.